scripts/deploy-v1.sh

```diff
@@ -75,7 +75,7 @@ gcloud compute backend-services add-backend cloud-deployment-gke-golang \
 # Ouput message for successful deployment
 echo -e "\n--------------------------------------------------------- "
 # Get Cloud Load Balancer configs
-FORWARDING_RULE_NAME="cloud-deployment-gke-golang"
+FORWARDING_RULE_NAME="cloud-deployment-gke-golang-fr"
```

Comment:
**Praise:** I like all the resource renames (adding the suffixes like `-cluster` and `-neg`) (instead of naming everything `cloud-deployment-gke-golang`).

Reply:
Thanks. This isn't my code originally, so it got confusing following so many similarly named resources while troubleshooting.  I liked the changes enough, I decided to keep them. 👍